src/vivarium_gates_iv_iron/components/hemoglobin.py

```diff
@@ -204,3 +179,13 @@ def disability_weight(self, index: pd.Index) -> pd.Series:
             name="anemia_levels",
         )
         return anemia_levels.map(ANEMIA_DISABILITY_WEIGHTS)
+
+    def _get_location_weights(self, builder: Builder):
+        if metadata.USE_PLW_LOCATION_WEIGHTS:
```

Comment:
put this in the config file, not in metadata.py

Reply:
It has been added as a population configuration.